tests/x11regressions/gnomecase/nautilus_permission.pm

```diff
@@ -14,13 +14,20 @@
 use base "x11regressiontest";
 use strict;
 use testapi;
+use version_utils;
```

Comment:
maybe you could only reduce to the needed functions, like this: `use version_utils qw(is_sle sle_version_at_least is_leap is_leap_version_at_least is_tumbleweed);`.

I don't know if there is a real rule about this, but lot of test developers do this :) I think it could help to reduce memory footprint or maybe test loading time.

Other than this, looks good.

Reply:
Fixed.